lib/llm-events/langchain/chat-completion-message.js

```diff
@@ -36,6 +36,12 @@ class LangChainCompletionMesssage extends LangChainEvent {
     params = Object.assign({}, defaultParams, params)
     super(params)
 
+    if (params.runId) {
```

Comment:
sorry for the back and forth here. let's defer this for a different ticket.  I spoke with uma on python agent and I think ids for LlmChatCompletionMessage are undefined unless passed in via `metadata.message_ids` It doesn't make much sense to me and don't want to hold this up. we can update later

Reply:
this is the code 

```py
def get_message_ids_add_nr_completion_id(run_args, completion_id):
    # invoke has an argument named "config" that contains metadata and tags.
    # Pop the message_ids provided by the customer off the metadata.
    # Add the nr_completion_id into the metadata to be used as the function call
    # identifier when grabbing the run_id off the transaction.
    metadata = (run_args.get("config") or {}).get("metadata") or {}
    message_ids = metadata.pop("message_ids", [])
    metadata["nr_completion_id"] = completion_id
    if not run_args["config"]:
        run_args["config"] = {"metadata": metadata}
    else:
        run_args["config"]["metadata"] = metadata
    return message_ids
``` 
but it raises more questions not worth figuring out in this PR

Reply:
completion_id is created elsewhere as a uuid so what we're doing for the summary is fine.

Reply:
Sounds good. But to hopefully clarify a little bit, the `runId` comes from https://api.js.langchain.com/classes/langchain_core_callbacks_base.BaseCallbackHandler.html#handleLLMStart